### ISSUE TITLE
No mixin [NoInterfaceObject] usage

### ISSUE THREAD

**Issue by annevk** [heycam/webidl#430]
Per https://github.com/heycam/webidl/pull/423#issuecomment-326332460 this exists. I'd like to study such usage and see if we can get rid of it.

**Comment by annevk** [comment]
Thanks, seems like we could just drop the keyword from those and be done with it. I guess part of the problem is some of those specifications not being maintained, but that can be fixed one way or another.

**Comment by tobie** [comment]
Well, we'd want to make sure suddenly exposing these interfaces to the Web doesn't break stuff, no?

**Comment by annevk** [comment]
Yeah, I suppose the ones with overly generic names should be prefixed, but seems safe otherwise.

**Comment by tobie** [comment]
Prefixing is still an observable change, too. No idea if it would have real impact, though.

**Comment by annevk** [comment]
How is it observable?

**Comment by tobie** [comment]
```js
navigator.geolocation.getCurrentPosition(g => console.log(Object.prototype.toString.call(g)));
```

**Comment by annevk** [comment]
Sigh. Part of the reason we had [NoInterfaceObject] was the ability to ship objects and change the name later. I guess that's another reason we should really get rid of it (and decide on names we can live with upfront).

**Comment by annevk** [comment]
WebGL seems fixed. I filed issues against two other specifications I identified from the list above. Anything else?

**Comment by domenic** [comment]
Searching through Blink, `External` in the HTML spec needs fixing. There's also https://github.com/w3c/mediacapture-main/issues/501 which is weird.

**Comment by domenic** [comment]
Also https://w3c.github.io/webappsec-feature-policy/#featurepolicy ... a little hard to tell since Blink doesn't have interface mixin syntax yet, so lots of stuff is mixed together.

**Comment by bzbarsky** [comment]
Fwiw, that's why I haven't looked at this in Gecko either.  It'll be a lot simpler once we implement mixin syntax...
